buildenv/jenkins/common/build

```diff
@@ -291,7 +291,14 @@ def archive() {
             sh "tar -C ${buildDir} -zcvf ${SDK_FILENAME} ${JDK_FOLDER}"
             // test if the test natives directory is present, only in JDK11+
             if (fileExists("${buildDir}${testDir}")) {
-               sh "tar -C ${buildDir} -zcvf ${TEST_FILENAME} ${testDir} --transform 's,${testDir},native-test-libs,'"
+                if (!SPEC.startsWith('zos')) {
```

Comment:
Can you invert this so we the condition is positive?

Reply:
done